erpnext_pk/install.py

```diff
@@ -0,0 +1,51 @@
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields
+
+
+
+custom_fields = {
+	# Doctype Company Mods
+	'Company': [
+		# NTN field
+		{"label": "NTN", "fieldname": "tax_ntn", "fieldtype": "Data", "insert_after": "tax_id"},
+		# STRN field
+		{"label": "STRN", "fieldname": "tax_strn", "fieldtype": "Data", "insert_after": "tax_ntn"}
+	],
+
+	# Doctype Customer Mods
+	'Customer': [
+		# NTN field
+		{"label": "NTN", "fieldname": "tax_ntn", "fieldtype": "Data", "insert_after": "tax_category"},
+		# CNIC field
+		{"label": "CNIC", "fieldname": "tax_cnic", "fieldtype": "Data", "insert_after": "tax_ntn"},
```

Comment:
CNIC field should be NIC, same way as NADRA calls it. The label can be CNIC for ease of understanding, however eventually it will change to Smart SNIC and later something else. So keep it NIC.

Reply:
That makes sense, will refactor later